src/tdastro/sources/physical_model.py

```diff
@@ -269,7 +269,7 @@ def compute_flux_with_extrapolation(self, times, wavelengths, graph_state, **kwa
 
         # If no extrapolation is needed, just call compute the flux.
```

Comment:
```suggestion
        # If no extrapolation is needed, just call compute SED.
```
or "just ~~call~~ compute the flux"